config/locales/en.yml

```diff
@@ -102,6 +102,7 @@ en:
           regional_delivery_officer: Regional delivery officer
           unassigned: Not yet assigned
           advisory_board_date: Date of advisory board
+          sharepoint_link: SharePoint link
```

Comment:
And here, I think it would be great to be super clear this is the establishment SharePoint folder, could we check with @SteveMilnes I guess we say 'school' not 'establishment'?